stable/rabbitmq-ha/README.md

```diff
@@ -256,7 +257,15 @@ the following keys:
 
 ### Prometheus Monitoring & Alerts
 
-Prometheus and its features can be enabled by setting `prometheus.enabled` to `true`.  See values.yaml for more details and configuration options
+As of RabbitMQ 3.8.0, it is possible to enable Prometheus metrics natively, no need to run an external exporter. To enable native Prometheus metrics, set `rabbitmqPrometheusPlugin.enabled` to `true`. This will expose all RabbitMQ node metrics via the `<<rabbitmqhost>>:15692/metrics` URL. Since all metrics are node local, they add the least pressure on RabbitMQ and will be available for as long as RabbitMQ is running, regardless of inter-node pressure or other nodes in the cluster going away.
+
+To learn more about RabbitMQ's native support for Prometheus, please refer to the official [Monitoring with Prometheus & Grafana guide](https://www.rabbitmq.com/prometheus.html).
+
+Team RabbitMQ manages Grafana dashboards that are meant to be used with the native Prometheus support. They are publicly available at [grafana.com/orgs/rabbitmq](https://grafana.com/orgs/rabbitmq).
+
+To enable metrics via the traditional rabbitmq_exporter, set `prometheus.enabled` to `true`. See `values.yaml` file for more details and configuration options.
+
+While there are two ways of monitoring RabbitMQ with Prometheus, both  `prometheus.enabled` and `rabbitmqPrometheusPlugin.enabled`can be enabled at the same time, they are not mutually exclusive. 
```

Comment:
Remove this line. Yes, technically they can be ran at the same time but my point about mutually exclusivity was that you wouldn't generally want to run both at the same time and it could cause incorrect metrics if they both update a metric with the same name.

Reply:
I removed the section, but let me say that native and HTTP are two different kinds of monitoring. 
>it could cause incorrect metrics 
I'd say  "incorrect reading"  instead of "incorrect metrics ":)! 
btw I agree with you, you have to know that native is for single node and http is for whole the cluster.

Reply:
Not sure what you mean by native vs HTTP. Native in the sense that it's built into RabbitMQ as a plugin, I guess. But they're both HTTP endpoints that Prometheus scrapes.

Reply:
Yes, native in this context means RabbitMQ plugin, which runs within the same system process as RabbitMQ, the Erlang VM.

Reply:
rabbitmq_exporter gets the metrics from the RabbitMQ Management HTTP API, and then serves them from its own TCP listener.

rabbitmq-prometheus, the plugin, runs and reads metrics directly from within the Erlang VM where RabbitMQ runs, and exposes them via a dedicated TCP listener. This TCP listener is owned by the same system process that runs RabbitMQ - it's the same Erlang VM. All this is wrapped in the `native` claim.

Would you agree @brian-brazil?

Reply:
That sounds about right to me. Using both shouldn't cause a problem, though I'd probably clarify that you usually only want to scrape one of them.